docs/Middleware.md

```diff
@@ -1,53 +0,0 @@
-<h1 align="center">Fastify</h1>
```

Comment:
There are links to this document, they should be removed.

Reply:
Okay